core/src/test/java/org/everit/json/schema/CombinedSchemaTest.java

```diff
@@ -55,17 +56,17 @@ public void anyCriterionSuccess() {
 
     @Test(expected = ValidationException.class)
     public void anyOfInvalid() {
-        CombinedSchema.anyOf(asList(
+        CombinedSchema.anyOf(Lists.<Schema>newArrayList(
                 StringSchema.builder().maxLength(2).build(),
                 StringSchema.builder().minLength(4).build()))
                 .build().validate("foo");
     }
 
     @Test
     public void factories() {
-        CombinedSchema.allOf(asList(BooleanSchema.INSTANCE));
-        CombinedSchema.anyOf(asList(BooleanSchema.INSTANCE));
-        CombinedSchema.oneOf(asList(BooleanSchema.INSTANCE));
+        CombinedSchema.allOf(Lists.<Schema>newArrayList(BooleanSchema.INSTANCE));
+        CombinedSchema.anyOf(Lists.<Schema>newArrayList(BooleanSchema.INSTANCE));
+        CombinedSchema.oneOf(Lists.<Schema>newArrayList(BooleanSchema.INSTANCE));
```

Comment:
Java7 needs explicit type declarations, so the staticly imported method wasn't good. I just prefer `Lists` to Ąrrays`, so I used that.